onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -44,20 +46,33 @@ def _pre_optimize(model: ir.Model) -> ir.Model:
     return model
 
 
-def fuse_xformers(model: ir.Model) -> ir.Model:
+def fuse_xformers(model: ir.Model) -> Tuple[ir.Model, Dict[str, int]]:
+    """
+    Apply transformer-specific fusions to the given model.
+
+    Args:
+        model: The input ONNX model represented as an `ir.Model`.
+
+    Returns:
+        A tuple containing:
+        - The optimized `ir.Model` after applying transformer-specific fusions.
+        - A dictionary with a count of each of the fusions applied.
+    """
+    fusion_count = dict()
+
     model = _pre_optimize(model)
-    fuse_rms_normalization(model)
-    fuse_normalization(model)
-    fuse_rotary_embedding(model)
-    fuse_partial_rotary_embedding(model)
-    fuse_cos_sin_cache(model)
-    fuse_sdpa(model)
-    fuse_mha(model)
-    fuse_gelu(model)
+    fusion_count["rms_normalization"] = fuse_rms_normalization(model)
+    fusion_count["rms_and_skip_normalization"] = fuse_normalization(model)
```

Comment:
Why "rms_and_skip"? It should be either "skip_" or "skip_rms_" ? This reminds me: we should cleanup fuse_normalization so it doesn't apply rms-normalization, which is redundant in this setting.

Reply:
Cleaned up, and also add SkipLayerNorm fusion, I have them as separate fusion calls for now so we can isolate the counts for the different types of skip fusions.